backend/git_real/activity_log_creators.py

```diff
@@ -11,12 +11,21 @@
 
 def log_pr_merged(pull_request):
     event_type, _ = EventType.objects.get_or_create(name=PR_MERGED)
-    LogEntry.objects.get_or_create(
-        actor_user=None,  # we don't care who clicked the merge button. Maybe we'll fill this in later
+    match = LogEntry.objects.filter(
+        actor_user=pull_request.merged_by,
         effected_user=pull_request.user,
-        object_1=pull_request,
+        object_1_id=pull_request.pk,
         event_type=event_type,
-    )
+        timestamp__gte=timezone.now() - timezone.timedelta(minutes=2),
```

Comment:
The timestamp looks a bit funny. What is the same event gets emitted twice becuase of a bug. This can happen. If there is an error on our backend then we can retry webhook requests and that can cause duplicates. If the webhook gets accessed twice it should not make 2 log entries and the time should be accurate.

Looking at the data from the webhook, there is a merged_at timestamp. Can that be used?